src/util.ts

```diff
@@ -12,6 +14,11 @@ export const configPathFn = (): string => {
       if (process.env.XDG_CONFIG_HOME) {
         return join(process.env.XDG_CONFIG_HOME, REPLUGGED_FOLDER_NAME);
       }
+      if (realUser) {
+        // Get the home directory of the sudo user from /etc/passwd
+        const homeDir = execSync(`getent passwd ${realUser}`).toString('utf-8').split(':')[5];
```

Comment:
Needs some error handling. Maybe throw if this either returns nothing or the result doesn't exist on the system?

Reply:
If the result doesn't exist on the system then that's a problem with your system, but it could be empty so I should handle that